protocol.md

```diff
@@ -266,6 +290,7 @@ HTTP/1.1 204 No Content
 Tus-Resumable: 1.0.0
 Tus-Version: 1.0.0,0.2.2,0.2.1
 Tus-Max-Size: 1073741824
+Tus-Max-Chunk-Size: 524288000
```

Comment:
I assume the Min is left out on purpose here to indicate these are options, correct?